cypress/support/components/actionspage.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+/** Navigate to Actions Page */
+function NavigateTo() {
+  cy.get('a[href$="/actions"]')
+    .click();
+}
+/** Click on create a new action button */
+function CreateNew() {
```

Comment:
The function names should mimic what they actually do. This function clicks the create button so name it `clickCreate`